cmd/client/main.go

```diff
@@ -86,6 +87,10 @@ var (
 						Name:  "subscribe",
 						Usage: "subscribe to push notifications about the swap's status",
 					},
+					&cli.StringFlag{
+						Name:  "eth-asset",
+						Usage: "Ethereum ERC-20 token address to receive, or the zero address for regular ETH",
+					},
```

Comment:
I think we probably want to do this per-swap (ie. in an offer advertisement) as opposed to configuring the entire daemon for it